guides/doc-Release_Notes/topics/foreman-3.9.2.adoc

```diff
@@ -0,0 +1,24 @@
+= Foreman 3.9.2
+
+A full list of changes is available on https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1799[Redmine]
+
+== Foreman
+
+=== JavaScript stack
+
+* pass:[EditorView snapshots change because of classnames update] - https://projects.theforeman.org/issues/37026[#37026]
+
+=== Reporting
+
+* pass:[Getting "undefined method '#name' for NilClass::Jail (NilClass) (Safemode::NoMethodError)" error generating subscription report ] - https://projects.theforeman.org/issues/37016[#37016]
+
+=== Web Interface
+
+* pass:[Closing parent nav should also close child nav] - https://projects.theforeman.org/issues/37067[#37067]
+* pass:[Expanding a section should collapse other expanded sections] - https://projects.theforeman.org/issues/37025[#37025]
+
+== Packaging
+
+=== RPMs
+
+* pass:[rubygem-ipmitool is missing ipmitool dependency] - https://projects.theforeman.org/issues/37246[#37246]
```

Comment:
In a0069692e04370c4e90db29ff93ece68a12d7084 the installation of ipmitool was added. This is now redundant and can be dropped from the procedure.